source/support.html.erb

```diff
@@ -12,28 +12,28 @@ description: Help and support connecting to GovWifi
       </div>
       <main class="govuk-grid-column-two-thirds" role="main" id="main">
         <h1 class="govuk-heading-l">GovWifi support</h1>
-        <h2 class="govuk-heading-m">User support</h2>
+        <h2 class="govuk-heading-m">Get help connecting to GovWifi</h2>
         <p class="govuk-body">
-          Use the left-hand navigation bar to view our device specific instructions for <%= link_to "connecting to GovWifi",
+          Use the navigation bar to view our device specific instructions for <%= link_to "connecting to GovWifi",
           "/about-govwifi/connect-to-govwifi/", class: "govuk-link" %> for the first time.
         </p>
         <p class="govuk-body">
-          For additional support, please contact the IT support desk in your building. They will be able to check your device and also ensure the local network is working.
+          You can also contact IT support in your building. They can check your device and make sure the local network is working.
         </p>
-        <h4 class="govuk-heading-s">Username or password reminder</h4>
+        <h4 class="govuk-heading-s">Get a username or password reminder</h4>
         <p class="govuk-body">
-          If you need a GovWifi username or password reminder, please send a blank email from the email address you registered with to <%= link_to "signup@wifi.service.gov.uk", "mailto: signup@wifi.service.gov.uk", class: "govuk-link" %>.
+          If you need a GovWifi username or password reminder, send a blank email from the email address you registered with to <%= link_to "signup@wifi.service.gov.uk", "mailto: signup@wifi.service.gov.uk", class: "govuk-link" %>
         </p>
         <h4 class="govuk-heading-s">Check GovWifi service status</h4>
         <p class="govuk-body">
-          You can check the <%= link_to "status of our central authentication service", "https://status.wifi.service.gov.uk", class: "govuk-link" %>.
+          If you're having trouble connecting to GovWifi, check our <%= link_to "status page", "https://status.wifi.service.gov.uk", class: "govuk-link" %> to see if there are any problems.
         </p>
         <p class="govuk-body">
-          If our service is operating normally and you are still having difficulty connecting to GovWifi, please contact the IT support desk in your building.
+          If there are not any problems, contact IT support in the building you're in.
```

Comment:
suggest: 'If there are no problems' or maybe 'If there are no problems on the status page'

Reply:
👍